keyboards/pearlboards/zeuspad/readme.md

```diff
@@ -0,0 +1,15 @@
+# Zeuspad
+
+![Zeuspad](https://i.imgur.com/F657DWwh.png)
+
+Zeuspad is the perfect numpad complement to my TKL, Zeus. Standalone it is a glorious numpad with an encoder, oled and rgb.
+
+* Keyboard Maintainer: Koobaczech
+* Hardware Supported: Zeuspad
+* Availability: https://pearlboards.net/products/zeuspadl-coming-2022?variant=39642771718276
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```